workbench-core/example/app/src/generateRouter.ts

```diff
@@ -31,7 +31,7 @@ export function generateRouter(): Express {
 
   //Adding rate limiting
   const limiter = rateLimit({
-    windowMs: 15 * 60 * 1000, // 15 minutes
+    windowMs: 1 * 60 * 1000, // 1 minute
```

Comment:
Just a note, I updated these values in my [PR](https://github.com/aws-solutions/solution-spark-on-aws/pull/603/files#diff-3699683c20549d55e454f077a9cd2d6b67e4e1f1f68a41048cf6b8e1a8303ad6)

Reply:
I'll revert this change and defer to your values